core/include/detray/definitions/math.hpp

```diff
@@ -0,0 +1,27 @@
+/** Detray library, part of the ACTS project (R&D line)
+ *
+ * (c) 2023 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+namespace detray {
+
+// SYCL include(s).
+#if defined(CL_SYCL_LANGUAGE_VERSION) || defined(SYCL_LANGUAGE_VERSION)
+#include <CL/sycl.hpp>
+#endif
+
+// System include(s).
+#include <cmath>
+
+/// Namespace to pick up math functions from
+#if defined(CL_SYCL_LANGUAGE_VERSION) || defined(SYCL_LANGUAGE_VERSION)
+namespace math_ns = cl::sycl;
```

Comment:
Can we maybe name the namespace somwhow differently ?

Reply:
Any suggestions? I just copied the code from algebra-plugins

Reply:
Hm, maybe ```detray::math``` or ```dmath```?

Reply:
Hmm I don't like both. let's just keep it as `math_ns`

Reply:
none of the namespaces in detray are marked with ```_ns```...